commons/src/main/java/org/infinispan/commons/util/concurrent/jdk8backported/AbstractEntrySizeCalculatorHelper.java

```diff
@@ -0,0 +1,18 @@
+package org.infinispan.commons.util.concurrent.jdk8backported;
+
+/**
+ * Abstract class that provides a method to round up to the nearest value of 8 which is important for most jvm when
+ * doing size calculations
+ * {@inheritDoc}
+ */
+public abstract class AbstractEntrySizeCalculatorHelper<K, V> implements EntrySizeCalculator<K, V> {
+   // This is how large the object header info is
+   public final static int OBJECT_SIZE = sun.misc.Unsafe.ADDRESS_SIZE;
```

Comment:
I believe order of modifiers should be `public static final` according to our coding conventions.